presto-orc/src/test/java/com/facebook/presto/orc/AbstractTestOrcReader.java

```diff
@@ -99,6 +99,13 @@ public void testLongSequence()
         testRoundTripNumeric(intsBetween(0, 31_234));
     }
 
+    @Test
+    public void testNegativeLongSequence()
+            throws Exception
+    {
+        testRoundTripNumeric(intsBetween(-31_234, -1000));
```

Comment:
Change this to -999. Also please add a comment to explain why it's -999 instead of 0.